video/out/wayland_common.c

```diff
@@ -2482,8 +2471,8 @@ static int set_colorspace(struct vo_wayland_state *wl)
         return -1;
     }
 
-    xx_image_description_creator_params_v4_set_primaries_named(wl->image_creator_params, xx_primaries);
-    xx_image_description_creator_params_v4_set_tf_named(wl->image_creator_params, xx_transfer);
+    wp_image_description_creator_params_v1_set_primaries_named(wl->image_creator_params, xx_primaries);
+    wp_image_description_creator_params_v1_set_tf_named(wl->image_creator_params, xx_transfer);
```

Comment:
Seems like the primaries/TF checks don't work above. mpv crashes instead of bailing out if the compositor doesn't support these.

(Also the variables should probably be renamed to drop "xx".)